azure-pipelines.yml

```diff
@@ -29,6 +29,7 @@ variables:
   NODE_VERSION: '10.14.1'
   YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
   BLOB_CONTAINER_NAME: '$web'
+  REACT_APP_API_URL: 'https://api.io.italia.it'
```

Comment:
@BurnedMarshal I moved `REACT_APP_API_URL` to root level because `azure-templates/deploy-steps.yml` contains an implicit build step